stl_text/datamodule/doc_classification.py

```diff
@@ -9,12 +9,34 @@
 from stl_text.ops.transforms import LabelTransform
 from torch.nn.utils.rnn import pad_sequence
 from stl_text.ops.samplers import PoolBatchSampler
+from iopath.common.file_io import PathManager
+
+from torchtext.experimental.transforms import (
+    sentencepiece_tokenizer,
+    sentencepiece_processor,
+    PRETRAINED_SP_MODEL,
+    TextSequentialTransforms,
+)
+from torchtext.experimental.vocab import (
+    build_vocab_from_text_file,
```

Comment:
Do we use `build_vocab_from_text_file` func in the file?

Reply:
> Do we use `build_vocab_from_text_file` func in the file?

Not really now anymore. I guess we do not not need this since sentencepiece is already pre-trained. As you suggested previously, we could directly use sentencepiece_processor to get token ids directly from input text. I just l just cleaned up code a bit.